packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -245,7 +245,8 @@ export default async function setupAndLink(
       project.name,
       org.slug,
       successEmoji,
-      autoConfirm
+      autoConfirm,
+      false // don't prompt to pull env for newly created projects
```

Comment:
We should refactor this in the future to use an object as the param so we can have more context on what this false connects to at the call site. But this looks good for now.